sdk/tables/azure-data-tables/azure/data/tables/aio/_table_service_client_async.py

```diff
@@ -262,8 +276,11 @@ def query_tables(
             page_iterator_class=TablePropertiesPaged
         )
 
-    def get_table_client(self, table, **kwargs):
-        # type: (Union[TableProperties, str], Optional[Any]) -> TableClient
+    def get_table_client(
+            self, table_name, # type: Union[TableProperties, str]
```

Comment:
it looks like this only takes a `table_name` now. does the type hint need to be updated?

Reply:
updated!